src/payment/payments.mock.ts

```diff
@@ -35,6 +35,15 @@ export function getCreditCardInstrument(): CreditCardInstrument {
     };
 }
 
+export function getVaultedInstrument(): VaultedInstrument {
+    return {
+        ccCvv: undefined,
+        ccNumber: '',
```

Comment:
Why the empty string? Since the value is typed as optional, it should either be `undefine` or the actual value. Unless that's what you are seeing in the API in which case we might need to adjust the types.